backend/prisma/migrations/20230701124358_reverse_share_options/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+CREATE TABLE "ReverseShareOptions" (
+    "id" TEXT NOT NULL PRIMARY KEY,
+    "shareId" TEXT NOT NULL,
+    "easyMode" BOOLEAN NOT NULL DEFAULT false,
```

Comment:
What exactly is the `easyMode`?  Is it the same as setting all other properties to false?